ui-tests/src/test/java/org/openmrs/reference/LoginTest.java

```diff
@@ -9,22 +9,19 @@
  */
 package org.openmrs.reference;
 
-import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.openmrs.reference.page.HomePage;
 
 import static org.junit.Assert.assertTrue;
 
-@Ignore
 public class LoginTest extends ReferenceApplicationTestBase {
     private HomePage homePage;
 
     @Test
     @Category(org.openmrs.reference.groups.BuildTests.class)
```

Comment:
this can be changed to `@Category(BuildTests.class)`

Reply:
Resolved thanks